docs: How to fix `unable to get local issuer certificate` error (#4553)
apache/apisix: docs/en/latest/FAQ.md

  "url": "http://127.0.0.1/get"
}
```

## How to fix `unable to get local issuer certificate` error

`conf/config.yaml`

```yaml
# ...
apisix:
  ssl:
    ssl_trusted_certificate: /path/to/certs/ca-certificates.crt
# ...
```

**Note:**

> - When you try to connect TLS services with cosocket should be set `apisix.ssl.ssl_trusted_certificate`

Reviewer: ```suggestion
- Whenever trying to connect TLS services with cosocket, you should set `apisix.ssl.ssl_trusted_certificate`
```

> Ref [@spacewander](https://github.com/spacewander)'s [issuecomment](https://github.com/apache/apisix/issues/4514#issuecomment-875377961) 

Reviewer 2: What about adding the quotes?

Author: Modified
